### DISCUSSION TITLE
How to import CalDAV on MacOS?

### DISCUSSION THREAD

**Discussion by Falci** [monicahq/monica#7179] [Q&A]
I'm not sure how to fill this form, specifically for the server path. I tried multiple things (/dav, /dav/calendars, /dav/calendars/<email>...) but no success.

<img width="411" alt="image" src="https://github.com/monicahq/monica/assets/1330573/55314388-e299-4517-ac56-fa02ebb6b28e">

**Comment by Falci** [comment]
Any help here?

**Comment by despokd** [comment]
[quinndiggity](https://github.com/quinndiggity) used in #6183 `/dav`:

> `curl -si https://app.monicahq.com/dav | grep -i 'www\-authenticate'`

I cannot offer more help since I don't have set up DAV. Also, I am not sure if DAV is already working in v5 ([Chandler](https://beta.monicahq.com/)). If you are using v4 it should be working.

Don't expect response from the owners currently: https://github.com/monicahq/monica/issues/6626#issuecomment-2002484416
